README.md

```diff
@@ -37,6 +39,8 @@ Deploy to your appengine instance OR run the AppEngineLauncher locally.
 
 1. If you see errors about missing the default Cloud Storage bucket, follow the [cloud integration instructions](https://cloud.google.com/appengine/docs/php/googlestorage/setup) to create a default bucket for your project.
 
+1. As a last resort, you can *clear the cache* by issuing a **GET** request to `https://YOUR_PROJECT_NAME.appspot.com/clear_cache.php`. You will be promoted for authentication, and then the script will delete symfony's cache from your Cloud Storage Bucket.
```

Comment:
> As a last resort
I got an impression that it will fix every issues, but I don't assume it's true. Can you reword?